koku/api/organizations/queries.py

```diff
@@ -273,12 +264,16 @@ def get_org_units(self):
                 org_unit_query = org_unit_query.exclude(deleted_timestamp__lte=self.start_datetime)
                 org_unit_query = org_unit_query.exclude(created_timestamp__gte=self.end_datetime)
                 val_list = [org_id, org_name, org_path, level]
+                org_unit_query = org_unit_query.order_by(f"{org_id}", f"-{created_field}").distinct(f"{org_id}")
+                org_ids = org_unit_query.values_list("id", flat=True)
+                org_id_list.extend(org_ids)
```

Comment:
I think that we will want to filter these org_ids based on the ones that we have access to so that a user with restricted access cannot see orgs inside of the `sub_orgs` list that they shouldn't.